python/tk_desktop/desktop_window.py

```diff
@@ -469,7 +473,9 @@ def _load_settings(self):
         self.set_on_top(self._settings_manager.retrieve("on_top", False))
 
         # always start pinned and hidden
```

Comment:
```suggestion
        # always start unpinned and shown
```
Should the comment be this instead?